apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/testing/vulnerability_report/VulnerabilityReport.jsx

```diff
@@ -140,6 +140,8 @@ const VulnerabilityReport = () => {
                 })
                 issuesFilters.filterCollectionsId = issuesFilters.filterCollectionsId.filter((x) => x !== undefined)
 
+                setCurrentDate(func.getFormattedDate(Date.now()/1000))
```

Comment:
this is not what we need

Reply:
this current date specifically or the entire thing ?